### ISSUE TITLE
How are emojis handled? How are they encoded?

### ISSUE THREAD

**Issue by sumegha19** [bfelbo/DeepMoji#58]
Hi https://github.com/bfelbo
Sorry for pestering you!  I am currently building a similar model for Emoji prediction for Hindi(India) language. I couldn't understand how are you treating emojis gathered from a tweet for training the model. If say X_train are the sentences from tweet data then what is Y_train? How are emojis handled? Are the one-hot encoded or some other methodology has been adopted?
Hoping for a response as I am a bit stuck here. I have written scripts for building vocabulary from the corpus gathered.
Thanks a ton !!

**Comment by bfelbo** [comment]
Hi. The top 64 emojis are one-hot-encoded and used as the labels. The remaining emojis are discarded (if I remember correctly). In the case of multiple emojis in a single tweet, we create a tweet in the training set for each unique emoji.

I hope this helps. There’s more details in the paper and I think there’s also a recording of my EMNLP presentation somewhere, which has an example :)
